src/actions.js

```diff
@@ -1,68 +1,71 @@
+/* @flow */
+import type { ActionType, PropertyType, ValueType } from './types';
+
 export const SET_ONION_FORM_FIELD_PROPERTY = 'SET_ONION_FORM_FIELD_PROPERTY';
 export const SET_ONION_FORM_MULTIPLE_FIELDS = 'SET_ONION_FORM_MULTIPLE_FIELDS';
 export const REGISTER_ONION_FORM_FIELD = 'REGISTER_ONION_FORM_FIELD';
 export const CLEAR_ONION_FORM_PROPERTY = 'CLEAR_ONION_FORM_PROPERTY';
 export const CLEAR_ONION_FORM = 'CLEAR_ONION_FORM';
 
-export function registerField(form, field) {
+export function registerField(form: string, field: string): ActionType {
   return {
-    type: REGISTER_ONION_FORM_FIELD,
+    type: 'REGISTER_ONION_FORM_FIELD',
     form,
     field,
   };
 }
 
-export function setMultipleFields(form, property, values) {
+export function setMultipleFields(form: string, property: PropertyType, values: Array<ValueType>): ActionType {
   return {
-    type: SET_ONION_FORM_MULTIPLE_FIELDS,
+    type: 'SET_ONION_FORM_MULTIPLE_FIELDS',
     form,
     property,
-    values
+    values,
   };
 }
 
-export function setFormFieldProperty(form, field, property, value) {
+export function setFormFieldProperty(form: string, field: string, property: PropertyType, value: ValueType): ActionType {
   return {
-    type: SET_ONION_FORM_FIELD_PROPERTY,
+    type: 'SET_ONION_FORM_FIELD_PROPERTY',
```

Comment:
Check for typos in  action type are covered by Flow. But for typing actions there are two different approaches, at least: https://blog.callstack.io/typed-redux-2aa8bff926ff#592d